Documentation/testing.md

```diff
@@ -49,16 +49,21 @@ valid RDMA resources for proper functioning.
 - traffic() runs <num> iterations of send/recv between 2 players.
 
 ## How to run rdma-core's tests
+#### Developers
+The tests can be executed from ./build/bin:
+./build.sh
+./build/bin/run_tests.py
+#### Users
 The tests are not a Python package, as such they can be found under
 /usr/share/doc/rdma-core-{version}/tests.
 In order to run all tests:
 ```
-cd /usr/share/doc/rdma-core-26.0/tests
-python3 run_tests.py
+python /usr/share/doc/rdma-core-<version>/tests/run_tests.py
 ```
+#### Execution output
 Output will be something like:
 ```
-$ python3 run_tests.py
+$ ./build/bin/run_tests.py
```

Comment:
Recent RHEL8 and CentOS8 show no longer python in the release image. How can we make it satisfy this line? Is there test setup requirements?

Reply:
From what I understand (link  below), /usr/bin/python has been removed from rhel8, thus the python alias. You should be able to use python3. Running from build/bin should be the same. Running from fresh installation will probably be something like:
python3 /usr/share/<>/run_tests.py

If your issue is different, please provide more details.

https://developers.redhat.com/blog/2019/05/07/what-no-python-in-red-hat-enterprise-linux-8/

Reply:
Also, python3 is defined as a requirement in RH's spec file of rdma-core.

Reply:
Ok, then this should not block the user. Thanks.